x-pack/dev-tools/chromium_version.js

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+require('../../src/setup_node_env');
+require('../build_chromium/chromium_version');
```

Comment:
It doesn't really matter too much to me, but any reason not to put this in the root scripts dir?

Reply:
This PR is x-pack specific because it deals with Reporting. I wasn't sure it fits well in the root scripts folder. Do you still think it would be a good fit?

Reply:
Sure I think the root scripts dir is good, I could also imagine `x-pack/plugins/reporting/scripts`

Reply:
8d53001c0b0